quilt/tools/check_functions.py

```diff
@@ -0,0 +1,103 @@
+import re
+from pandas import DataFrame as df
+import numpy
+import pytest
+
+filename = None
```

Comment:
Globals?

Reply:
COMMENTED - needed for checks: syntax...